src/data_tools/include/data_tools/jsf_data.h

```diff
@@ -58,13 +61,53 @@ struct jsf_sss_ping
 cv::Mat make_waterfall_image(const jsf_sss_ping::PingsT& pings);
 void show_waterfall_image(const jsf_sss_ping::PingsT& pings);
 
+
+struct dvl_reading
```

Comment:
Please rename this `jsf_dvl_ping`

Reply:
Okay and I will rename the "reading" to "ping" as well.